charts/fybrik/values.yaml

```diff
@@ -191,6 +191,15 @@ manager:
 
   prometheus: false
 
+  # CSP solver for data plane optimization
+  solver:
+    # image of the container with solver binary and libs
+    # when specified, the solver will be deployed in the manager pod
+    image: "optimizer"
+    imagePullPolicy: "Always"
+    # Set to true to enable the use of the solver by Fybrik
+    enabled: false
```

Comment:
The default is to deploy the solver but to have it turned off?  Shouldn't the default be to have it turned on?

Reply:
I still consider this as an experimental release. If it goes into 0.7 it should definitely be "false". Later we'll change it to "true", after more runs and features.

Reply:
OK.  Makes sense